modules/tournament/src/main/TournamentScheduler.scala

```diff
@@ -509,6 +516,15 @@ final private class TournamentScheduler(
       ),
       scheduleYearly24hr(Variant.Go(strategygames.go.variant.Go9x9), Blitz32)(
         new DateTime(2024, 10, 4, 0, 0)
+      ),
+      scheduleYearly24hr(
+        Variant.FairySF(strategygames.fairysf.variant.BreakthroughTroyka),
+        Blitz32
+      )(
+        new DateTime(2024, 10, 11, 0, 0)
+      ),
+      scheduleYearly24hr(Variant.Draughts(strategygames.draughts.variant.Brazilian), Blitz32)(
+        new DateTime(2024, 10, 18, 0, 0)
```

Comment:
this is fine, but remember to make a note to delete the current Brazilian Yearly when we deploy, so we dont have two. I think you already have a note to rename current draughts breakthrough tournaments.